libs/wingc/src/type_check.rs

```diff
@@ -719,9 +719,16 @@ impl<'a> TypeChecker<'a> {
 				}
 
 				// Make sure this is a function signature type
-				let func_sig = func_type
-					.as_function_sig()
-					.expect(&format!("{} should be a function or method", function));
+				let func_sig = if let Some(func_sig) = func_type.as_function_sig() {
+					func_sig
+				} else {
+					self.expr_error(
+						exp,
+						format!("\"{}\" should be a function or method",function
```

Comment:
I suspect this isn't formatted using our rustfmt settings. We should try to make sure that all commits use the same fmt settings to avoid extra "non functional" commits in the future.
```suggestion
						format!("\"{}\" should be a function or method", function
```

Reply:
Thanks!